napari/layers/_markers_layer/view/properties.py

```diff
@@ -93,7 +93,8 @@ def changeSymbol(self, text):
         self.layer.symbol = text
 
     def changeSize(self, value):
-        self.layer.size = value
+        avg = np.mean(self.layer.size) or 1
```

Comment:
I think it would be nice to have a comment explaining what is being done here.

Reply:
doc string added